From 84195d1efab9720f7d378cd4e01a57eb5b864a2e Mon Sep 17 00:00:00 2001
From: Kieran Bingham <kieran.bingham@ideasonboard.com>
Date: Mon, 18 Mar 2019 17:17:40 +0000
Subject: [PATCH 376/703] vcsm: Reduce scope of local functions

The functions:

  vc_vchi_sm_send_msg
  vc_sm_ioctl_alloc
  vc_sm_ioctl_alloc_share
  vc_sm_ioctl_import_dmabuf

Are declared without a prototype. They are not used outside of this
module, thus - convert them to static functions.

Signed-off-by: Kieran Bingham <kieran.bingham@ideasonboard.com>
---
 drivers/char/broadcom/vc_sm/vc_vchi_sm.c |  2 +-
 drivers/char/broadcom/vc_sm/vmcs_sm.c    | 14 +++++++-------
 2 files changed, 8 insertions(+), 8 deletions(-)

--- a/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
+++ b/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
@@ -375,7 +375,7 @@ lock:
 	return -EINVAL;
 }
 
-int vc_vchi_sm_send_msg(struct sm_instance *handle,
+static int vc_vchi_sm_send_msg(struct sm_instance *handle,
 			enum vc_sm_msg_type msg_id,
 			void *msg, uint32_t msg_size,
 			void *result, uint32_t result_size,
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -1574,8 +1574,8 @@ error:
 }
 
 /* Allocate a shared memory handle and block. */
-int vc_sm_ioctl_alloc(struct sm_priv_data_t *private,
-		      struct vmcs_sm_ioctl_alloc *ioparam)
+static int vc_sm_ioctl_alloc(struct sm_priv_data_t *private,
+			     struct vmcs_sm_ioctl_alloc *ioparam)
 {
 	int ret = 0;
 	int status;
@@ -1685,8 +1685,8 @@ error:
 }
 
 /* Share an allocate memory handle and block.*/
-int vc_sm_ioctl_alloc_share(struct sm_priv_data_t *private,
-			    struct vmcs_sm_ioctl_alloc_share *ioparam)
+static int vc_sm_ioctl_alloc_share(struct sm_priv_data_t *private,
+				   struct vmcs_sm_ioctl_alloc_share *ioparam)
 {
 	struct sm_resource_t *resource, *shared_resource;
 	int ret = 0;
@@ -2200,9 +2200,9 @@ error:
 }
 
 /* Import a contiguous block of memory to be shared with VC. */
-int vc_sm_ioctl_import_dmabuf(struct sm_priv_data_t *private,
-			      struct vmcs_sm_ioctl_import_dmabuf *ioparam,
-			      struct dma_buf *src_dma_buf)
+static int vc_sm_ioctl_import_dmabuf(struct sm_priv_data_t *private,
+				     struct vmcs_sm_ioctl_import_dmabuf *ioparam,
+				     struct dma_buf *src_dma_buf)
 {
 	int ret = 0;
 	int status;
